### ISSUE TITLE
Android | Some itens of the screen are getting black when running on device.

### ISSUE THREAD

**Issue by TBrayner** [InvexGames/MaterialUI#53]
![screenshot_2015-10-12-22-31-38](https://cloud.githubusercontent.com/assets/3511497/10444134/76f24568-7195-11e5-91a0-37a9540fb60e.png)

I really don't know what is happening.. I downloaded several times the package and unintalled Unity 2 times...  I'm not doing anything beyond to create the appbar.

**Comment by DecSmith42** [comment]
Are you using Unity 5.2 by any chance?

**Comment by TBrayner** [comment]
Yes, I'm using the last version of unity. I downloaded it just few days ago.

**Comment by DecSmith42** [comment]
Unity 5.2 (and 5.3) brought with it a few large changes to how UI is done (and also still contains some rare but weird UI bugs), and as this version of MaterialUI is no longer being maintained, it's entirely possible that it just won't work properly.

I'd try and have a look at it, but I'm currently overwhelmed with working on the new MaterialUI, other projects, and university assessment.

However, if you manage to identify the problem, I'll gladly take pull requests :)

**Comment by TBrayner** [comment]
And do yoiu have an idea when you will release the AssetStore kit?

**Comment by DecSmith42** [comment]
Soon™

Serious answer: At a guess, a few weeks, maybe more. I'm planning on definitely having it out by the end of November. It heavily depends on how quickly I can get through my uni work, but once my semester's over (~end of this month), I'll be able to work on MaterialUI almost full-time.

**Comment by TBrayner** [comment]
Nice, I'm starting a new project, and this kit could help me a lot on its interface, I'll wait for the Asset release.
 thks

**Comment by DecSmith42** [comment]
You won't be disappointed :)

**Comment by EnderPicture** [comment]
So... Now it is 2016, is MaterialUI released on the Unity Asset Store? If so, link?

**Comment by nah0y** [comment]
Hey EnderPicture,
We are almost ready for a first version that we'll send to the Asset Store, hopefully this week!

After that, we'll have to wait for them to approve the package.
